src/server/services/wms/qgswmsgetcapabilities.cpp

```diff
@@ -680,6 +681,23 @@ namespace QgsWms
       composerTemplateElem.setAttribute( QStringLiteral( "width" ), width.length() );
       composerTemplateElem.setAttribute( QStringLiteral( "height" ), height.length() );
 
+      //atlas enabled and atlas covering layer
+      QgsLayoutAtlas *atlas = layout->atlas();
+      if ( atlas && atlas->enabled() )
+      {
+        composerTemplateElem.setAttribute( QStringLiteral( "atlasEnabled" ), QStringLiteral( "1" ) );
+        QgsVectorLayer *cLayer = atlas->coverageLayer();
+        if ( cLayer )
+        {
+          QString layerName = cLayer->shortName();
+          if ( layerName.isEmpty() )
+          {
+            layerName = cLayer->name();
+          }
```

Comment:
I think that the `layerName` could be the id of the layer if `QgsServerProjectUtils::wmsUseLayerIds( *project )` is `True`.